factorio-map-process.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
```

Comment:
Thoughts on this instead?
```suggestion
#!/usr/bin/env bash
```